src/docs/product/sentry-basics/grouping-and-fingerprints/index.mdx

```diff
@@ -11,21 +11,21 @@ description: "Learn more about how Sentry groups issues together as well as diff
 Have you ever had a set of similar-looking issues like this?
 ![Issues Dashboard](issues-dashboard.png)
 
-Let's first understand how events group into an issue. The issue `testTypeIssue16` has 18 events that Sentry grouped into an issue because they share the same _fingerprint_.
+Let's first understand how errors and transactions group into an issue. For example, the issue `testTypeIssue16` has 18 events that Sentry grouped into an issue because they share the same _fingerprint_.
 
 **What is a fingerprint?**
 A fingerprint is a way to uniquely identify an event. Sentry sets a fingerprint by default for you. Sentry uses its own built-in grouping algorithms to generate a fingerprint based on information available within the event such as a stack trace, exception, and message. Events with the same fingerprint are grouped together.
 
-**Do I need to do anything?** No. A stack trace is used by default if it's available. So if two events have the same stack trace, then they will group into one issue. If you have separate issues that you'd like to group together, then this guide will show you how.
+**Do I need to do anything?** No. Sentry automatically generates a fingerprint based on the event type and all available information. If events have the same root cause, they will group into one issue. If you have separate issues that you'd like to group together, then this guide will show you how.
```

Comment:
Later we talk about the solutions only being for error issues, so I'm thinking we should update this?

```suggestion
**Do I need to do anything?** No. Sentry automatically generates a fingerprint based on the event type and all available information. If events have the same root cause, they will group into one issue. If you have separate error issues that you'd like to group together, then this guide will show you how.
```